scripts/ci/images/ci_prepare_prod_image_on_ci.sh

```diff
@@ -48,4 +48,61 @@ function build_prod_images_on_ci() {
 }
 
 
+
+function verify_prod_image_has_airflow_and_providers {
```

Comment:
Can you add a test that will check if at least one provider can be loaded? This will give us the greatest confidence that everything is working.
```
python -c "from airflow.providers.google.common.hooks.base_google import GoogleBaseHook"
```
We may have these files in the image, but for various reasons they may not be loaded by Python, e.g. due to invalid permissions.

Reply:
I can do better than that. I already have script that imports all of the providers (I am using that in providers package/backport providers package preparation and it is perfectly suited for that case. it is even very well prepared to handle this case.

Reply:
However it needs #12082 . So I'd rather merge it now and add the test in #12082 @mik-laj if that's ok?

Reply:
SGTM.